### DISCUSSION TITLE
It's pulling accounts from all OU's in active directory. Is there a way to omit certain OU's?

### DISCUSSION THREAD

**Discussion by gobarrium** [YannickRe/spfx-msgraph-peoplesearch#88] [Q&A]
So with the wide open search I'm seeing it pull in every AD OU out there, (e.g. disabled OU folders, service accounts, etc.)  Is there a way for it to filter only specific OU's so that people aren't seeing all the unnecessary OU's?

**Comment by YannickRe** [comment]
It uses the Microsoft Graph /users endpoint, and thus shows you everything by default. It's up to you to configure additional filters in the properties of the web part. You can try and create a filter that works for you in Graph Explorer, and then put it in the web part.
You can use the following for guidance: https://learn.microsoft.com/en-us/graph/api/user-list?view=graph-rest-1.0&tabs=http

**Comment by gobarrium** [comment]
Thank you so much for your response.  I've found that it looks like I can pull up accounts based on a certain assigned license. I did play with microsoft graph with the query. This would work for me. But I don't know how to get that query into the sharepoint peoplesearch webpart.  I didn't see anywhere I could interact with graph inside of the webpart editing.
    On Thursday, August 15, 2024 at 03:48:55 PM EDT, Yannick Reekmans ***@***.***> wrote:  
 
 


It uses the Microsoft Graph /users endpoint, and thus shows you everything by default. It's up to you to configure additional filters in the properties of the web part. You can try and create a filter that works for you in Graph Explorer, and then put it in the web part.
You can use the following for guidance: https://learn.microsoft.com/en-us/graph/api/user-list?view=graph-rest-1.0&tabs=http

**Comment by YannickRe** [comment]
You can put the filter part of your query in the $filter field of the web part properties:
![image](https://github.com/user-attachments/assets/17465f43-5db5-4d1e-b0ed-0d3ec0daa042)

**Comment by gobarrium** [comment]
Thanks you.  I'm unable to find that "$filter paramater value" option in your provided screen shot in any of the pages I've scrolled through when editing the web parts for this. Is there an option I need to turn on?
    On Friday, August 16, 2024 at 11:42:43 AM EDT, Yannick Reekmans ***@***.***> wrote:  
 
 


You can put the filter part of your query in the $filter field of the web part properties:
image.png (view on web)
